cabal.project

```diff
@@ -26,6 +26,8 @@ packages:
   sop-extras
   strict-sop-core
 
+allow-newer: plutus-core:cardano-crypto-class
```

Comment:
This is something that we will have to address for the 10.3 release, so I hope it disappears before we merge this PR.

Reply:
I hope so too, but it's not under my control. We need a newer `cardano-crypto-class`, so the only way to make that compatible with plutus is to upgrade the dependency there and then depend on the upgraded plutus ourselves.

Reply:
Actually, it looks like this is no longer needed.

Reply:
It is a bit complicated because of some dancing in dependency bounds (a revision expanded the bounds, a later revision restricted them again). Let's see after the release for 10.3 I think this will not be needed. (but I'm surprised you don't seem to need it anymore in this branch?)